assets/js/http.js

```diff
@@ -0,0 +1,14 @@
+class Http {
+  async get(url) {
+    const response = await fetch(url);
+
+    if (!response.ok) {
+      const message = `An error has occured: ${response.status}`;
+      throw new Error(message);
+    }
+
+    const data = await response.json();
+
+    return data;
+  }
+}
```

Comment:
Consider using a simple object notation -- since no state is used inside a class instance:
```
export const http = {
  async get(url) { ... }
}
```